custom/enikshay/integrations/ninetyninedots/api_spec.py

```diff
@@ -59,24 +94,170 @@ def load_api_spec():
     return spec
 
 
+class MermParams(StrictJsonObject):
+    IMEI = jsonobject.StringProperty(required=False, exclude_if_none=True)
+    daily_reminder_status = jsonobject.StringProperty(required=False, exclude_if_none=True)
+    daily_reminder_time = jsonobject.StringProperty(required=False, exclude_if_none=True)  # HH:mm
+    refill_reminder_status = jsonobject.StringProperty(required=False, exclude_if_none=True)
+    refill_reminder_datetime = jsonobject.StringProperty(
+        required=False,
+        exclude_if_none=True
+    )  # yy/MM/dd HH:mm:ss
+    RT_hours = jsonobject.StringProperty(
+        required=False,
+        exclude_if_none=True
+    )  # 1 = 12 hours; i.e. for 3 days - RT_hours = 6
+
+
+class BasePatientPayload(StrictJsonObject):
```

Comment:
Curious how you decided what to put in code (here) versus in the spec (yaml file)